images/router/haproxy/conf/haproxy-config.template

```diff
@@ -312,6 +308,9 @@ frontend public_ssl
 # traffic
 ##########################################################################
 backend be_sni
+        {{- with $value := maxTimeoutFirstMatchedAndClipped .State "haproxy.router.openshift.io/timeout" $timeSpecPattern (env "ROUTER_DEFAULT_SERVER_TIMEOUT") "30s"  }}
+  timeout server  {{ $value }}
```

Comment:
Was this annotation ever being applied before?

Reply:
I didn't understand which annotation you are referring to. `timeout server` was not set in `be_sni` before, it's needed to prevent [the warning message](https://github.com/openshift/router/pull/536#issuecomment-1834660723).